wiki/Beatmapping/Circle_size/en.md

```diff
@@ -0,0 +1,33 @@
+---
+tags:
+  - CS
+---
+
+# Circle size
+
+**Circle size** (***CS***) is a difficulty setting which determines the size of [hit objects](/wiki/Hit_Objects) on [beatmaps](/wiki/Beatmaps). The value for circle size ranges from 2 to 7, where a higher value will make hit objects smaller. Most beatmaps use a value between 3 and 5, while values between 5 and 6 are sometimes used. Values between 6 and 7 are almost never used. Circle size values higher than 7 or lower than 2 can be seen when the [Hard Rock](/wiki/Game_Modifiers#hard-rock) or [Easy](/wiki/Game_Modifiers#easy) mods are used respectively.
```

Comment:
saying it ranges from 2 to 7 seems wrong, cuz 0-10 are all valid too. if you want to mention that the song setup panel recommends 2-7 i think u should make that more clear

Reply:
agree